doc-Managing_Infrastructure_and_Inventory/topics/Detaching_EBS_Volumes.adoc

```diff
@@ -0,0 +1,11 @@
+[[detaching_ebs_volumes]]
```

Comment:
Hi @adahms - similar question to the one about Deleting an Volume -- is there a difference between this file (Detaching_EBS_Volumes.adoc) and Detaching_Volumes.adoc, and if so, should the language mention specifically the provider?